### DISCUSSION TITLE
How to change the Command Palette's font?

### DISCUSSION THREAD

**Discussion by huylv** [wezterm/wezterm#5289] [Q&A] [answered]
### What Operating System(s) are you running on?

Linux X11

### Which Wayland compositor or X11 Window manager(s) are you using?

Gnome 3

### WezTerm version

20240203-110809-5046fc22

### Ask your question!

I'm on Ubuntu 22.04 LTS. The default font on the command palette looks big and ugly. How can I change it?

**Comment by nekowinston** [comment, marked as answer]
via `config.window_frame`, e.g.:
```lua
local wezterm = require("wezterm")

local c = wezterm.config_builder()

c.font = wezterm.font_with_fallback({
	"Berkeley Mono",
	"Symbols Nerd Font",
})
c.font_size = 13
c.command_palette_font_size = c.font_size
c.window_frame = {
	font = wezterm.font("IBM Plex Sans"),
}

return c
```

**Reply by huylv** [reply]
@nekowinston: Thank you very much. It works for the most part, except the `font_size` in `c.window_frame` seems to have no effect.

Update: I found out that it can be adjusted with `command_palette_font_size`. Source: https://wezfurlong.org/wezterm/config/lua/config/command_palette_font_size.html

**Reply by nekowinston** [reply]
I've updated the example code, thanks for the correction.
